politeiawww/api/cms/v1/v1.go

```diff
@@ -60,12 +65,28 @@ const (
 	ContractorTypeDirect        ContractorTypeT = 1 // Direct contractor
```

Comment:
General comment about these `ContractorTypeT` and `DCCStatusT`, don't add types/statuses before they're needed.  If they're not being implemented this PR then they should be removed from the PR.  Put them in the PR in which they're being implemented.  If they're not being implemented in any of the PRs (ex. `DCCStatusDebate`) then they should be there at all.